.github/workflows/glogger.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
+        pip install --force-reinstall importlib_metadata~=4.8  # Added requirement separately since it stems from an errorneous lint. See the following link for more details: https://github.com/Granulate/granulate-utils/pull/82#discussion_r985745999
```

Comment:
So why separately and not in dev-requirements? And why --force-reinstall?

Reply:
Maybe the comment wasn't clear, `importlib_metadata==5` is installed as it is a requirement of 1 of the packages used by granulate-utils/gLogger. I need importlib_metadata~=4.8 so I must reinstall it in order for it to work. 

It is separate from dev-requirements.yml because it should be temporary until they fix the problematic requirements in the link suggested.

Reply:
I managed. Do the same fix here and remove the `importlib_metadata` thing